workflows/multiomics/integration/config.vsh.yaml

```diff
@@ -41,15 +35,31 @@ functionality:
           direction: output
           description: Destination path to the output.
           example: output.h5mu
+    - name: PCA options
+      arguments:
         - name: "--obsm_pca"
           type: string
           default: "X_pca"
           description: "In which .obsm slot to store the resulting PCA embedding."
+        - name: "--var_input_pca"
```

Comment:
I feel like `var_input_pca` is not an intuitive parameter name. What about `var_pca_feature_selection`?

Reply:
Adjusted!